downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -178,34 +176,38 @@ public static Integer assignDateForGOALine(GKInstance entityInst, String goaLine
 
     /**
      * Retrieves date from instance and formats it for GO annotation file.
-     * @param instanceEditInst
-     * @return
-     * @throws Exception
+     * @param instanceEditInst -- GKInstance, instanceEdit from either a Modified or Created instance.
+     * @return -- Integer, from instanceEdit's dateTime. Parsed to remove the Timestamp.
+     * @throws Exception -- MySQLAdaptor exception.
      */
     private static int getDate(GKInstance instanceEditInst) throws Exception {
         return Integer.valueOf(instanceEditInst.getAttributeValue(ReactomeJavaConstants.dateTime).toString().split(" ")[0].replaceAll("-", ""));
     }
 
     /**
      * Iterates through the lines in the 'goaLines' list, retrieves the date associated with that line and also adds the 'Reactome' column before adding it to the gene_association.reactome file.
-     * @param filename
-     * @throws IOException
+     * @throws IOException -- File writing/reading exceptions.
      */
-    public static void outputGOAFile(String filename) throws IOException {
+    public static void outputGOAFile() throws IOException {
 
-        Files.deleteIfExists(Paths.get(filename));
+        Files.deleteIfExists(Paths.get(GOAGeneratorConstants.GOA_FILENAME));
         List<String> sortedGoaLines = new ArrayList<>(goaLines);
         Collections.sort(sortedGoaLines);
-        BufferedWriter br = new BufferedWriter((new FileWriter(filename)));
+        BufferedWriter br = new BufferedWriter((new FileWriter(GOAGeneratorConstants.GOA_FILENAME)));
         br.write("!gaf-version: 2.1\n");
         for (String goaLine : sortedGoaLines) {
-            br.append(goaLine + "\t" + dates.get(goaLine) + "\tReactome\t\t\n");
+            br.append(goaLine + "\t" + dates.get(goaLine) + "\t" + GOAGeneratorConstants.REACTOME_STRING + "\t\t\n");
```

Comment:
You might be able to replace the loop writing the lines with a call to `Files.write(...)`.

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#write-java.nio.file.Path-java.lang.Iterable-java.nio.file.OpenOption...-